extensions/logging-sentry/runtime/src/main/java/io/quarkus/sentry/SentryConfig.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.sentry;
+
+import java.util.Optional;
+import java.util.logging.Level;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class SentryConfig {
+
+    /**
+     * Sentry DSN
+     *
+     * The DSN is the first and most important thing to configure because it tells the SDK where to send events. You can find
+     * your project’s DSN in the “Client Keys” section of your “Project Settings” in Sentry.
+     */
+    @ConfigItem
+    public String dsn;
+
+    /**
+     * The sentry log level.
+     */
+    @ConfigItem(defaultValue = "WARNING")
```

Comment:
We generally want to stick with the log4j style names I think, so this should be `WARN`.

Reply:
@dmlloyd I used `java.util.logging.Level` like in `ConsoleConfig` and it doesn't include `WARN`. What should I do?

Reply:
@dmlloyd by some magic (I guess `LevelConverter`) it works with `WARN`. This should be documented somewhere I think.